src/rust/engine/process_executor/src/main.rs

```diff
@@ -39,7 +39,7 @@ fn main() {
     (command, env)
   };
 
-  let result = process_executor::run_command(process_executor::ExecuteProcessRequest { argv, env }).unwrap();
+  let result = process_executor::local::run_command_locally(process_executor::ExecuteProcessRequest { argv, env }).unwrap();
```

Comment:
Take it or leave it, but: there might already be an advantage to adding a `trait Executor` (even with only one method), as it will help make it obvious that there should be uniformity between the two implementations moving forward.

Reply:
They're actually not quite compatible today (slightly different error types), and they're also not on any struct... I'll pull one out when there's actually a caller :)

Reply:
Ok, that's fine too!

For the record, it's still possible to implement a trait, as traits can have "static" (no `self` arg) methods... you can then use them generically via:
```
fn execute_thing<E: Executor>(cmd: ...) -> ... {
  E::execute(cmd)
}
```